File: ServicePr-master/ServicePr-master/app/src/main/java/kr/ac/kau/sw/a2016125063/servicepr/MainActivity.java
package kr.ac.kau.sw.a2016125063.servicepr;

import android.app.ActivityManager;
import android.content.Intent;
import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;
import android.widget.TextView;

public class MainActivity extends AppCompatActivity {

    private TextView usageText;

    private long accumulatedTime;

    private long min;

    private long sec;

    private Button serviceButton;


    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        min = accumulatedTime/60;
        sec = accumulatedTime%60;
        usageText = (TextView) findViewById(R.id.usageText);
        usageText.setText(getString(R.string.not_running));

        serviceButton = (Button) findViewById(R.id.serviceButton);
        serviceButton.setOnClickListener(new View.OnClickListener(){
            @Override
            public void onClick(View v){
                Intent intent = new Intent(getApplicationContext(),PracticeService2.class);
                intent.putExtra("accumulatedTime",accumulatedTime);
                startService(intent);
            }
        });

    }
}

File: UsageStatsManager/app/src/main/java/kr/ac/kau/sw/a2016125063/myapplication/MainActivity.kt
package kr.ac.kau.sw.a2016125063.myapplication

import android.support.v7.app.AppCompatActivity
import android.os.Bundle
import android.content.Context.APP_OPS_SERVICE
import android.R.attr.mode
import android.content.Intent
import android.provider.MediaStore
import kotlinx.android.synthetic.main.activity_main.*
import android.R.attr.data
import android.util.Log
import kr.ac.kau.sw.a2016125063.myapplication.R.id.*


class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

<<<<<<< HEAD
        cameraButton.setOnClickListener{
            intent = Intent(MediaStore.ACTION_IMAGE_CAPTURE)
=======
        val intent = Intent(applicationContext, UsageService::class.java)
        startService(intent)

        cameraButton.setOnClickListener(View.OnClickListener {
            val intent = Intent(MediaStore.ACTION_IMAGE_CAPTURE)
>>>>>>> b70ac9d3ecad3e085efe81e188caad661659558e
            startActivityForResult(intent,1)

        }

        serviceButton.setOnClickListener{
            Log.d("ServiceButton : ","OnClickListener")
            val intent = Intent(applicationContext, UsageService::class.java)
            startService(intent)
        }


        appLockServiceButton.setOnClickListener{
            Log.d("LockButton : ","OnClickListener")
            val intent = Intent(applicationContext, ActivityManagerService::class.java)
            startService(intent)
        }

    }

    override fun onActivityResult(requestCode: Int, resultCode: Int, data: Intent?) {
        super.onActivityResult(requestCode, resultCode, data)
        iv.setImageURI(data!!.data)
        
    }
}
